index.js

```diff
@@ -1,44 +1,57 @@
-var async = require('async')
-var get = require('lodash.get')
-var has = require('lodash.has')
-
-module.exports = function(options) {
-
-    var RabbitClient = get(options, 'RabbitClient') || require('rascal')
-    var components = get(options, 'components') || {}
-    var broker
-    var config
-    var logger
-
-    function init(dependencies, cb) {
-        config = dependencies.config
-        logger = dependencies.logger || console
-        cb()
-    }
+const { has, get } = require('lodash');
 
-    function validate(cb) {
-        if (!has(config, 'vhosts')) return cb(new Error('config.vhosts is required'))
-        cb()
-    }
+module.exports = options => {
 
-    function start(cb) {
-        logger.info('Connecting to rabbitmq')
-        var conf = RabbitClient.withDefaultConfig(config);
+  const RabbitClient = get(options, 'RabbitClient') || require('rascal');
```

Comment:
https://youmightnotneed.com/lodash

or 

https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore#_get

Reply:
You're right, let's get rid of it